nominees/fao-digital-services-portfolio-weather-&-crop-calendar.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "FAO Digital Services Portfolio - Weather & Crop Calendar",
+  "aliases": [
+    ""
```

Comment:
Same comment - @nathanbaleeta - Should we add an alias here, similar to the entry for FAMEWS?

Reply:
Resolved in commit 91da3ab